test/unittests/execute_call_test.cpp

```diff
@@ -694,14 +702,13 @@ TEST(execute_call, imported_function_from_another_module_max_depth)
     const auto func_idx = fizzy::find_exported_function(*instance1->module, "f");
     ASSERT_TRUE(func_idx.has_value());
 
-    auto sub = [&instance1, func_idx](Instance&, const Value* args, int depth) -> ExecutionResult {
-        return fizzy::execute(*instance1, *func_idx, args, depth + 1);
-    };
-
-    auto instance2 = instantiate(std::move(module2), {{sub, instance1->module->typesec[0]}});
+    auto instance2 =
+        instantiate(std::move(module2), {{{*instance1, *func_idx}, instance1->module->typesec[0]}});
```

Comment:
This actually changes what the test is doing (importing wasm function instead of host function), and hence expected result. I should change it back to using lambda.